src/components/_vaults/StrategyRow.tsx

```
@@ -0,0 +1,237 @@
+    : undefined
+
+  const isPreLaunch = useMemo(() => {
+    return !!launchDate && Date.now() < launchDate.getTime()
+  }, [launchDate?.getTime?.()])
+
+  // User net value (reuse manage page logic via hook)
+  const strategyAddress = vault?.config?.cellar?.address
+  const strategyChainId = vault?.config?.chain?.id
+  const { data: userStratData } =
+    strategyAddress && strategyChainId
+      ? useUserStrategyData(strategyAddress, strategyChainId)
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)